packages/blade/src/components/DatePicker/CalendarStyles.web.tsx

```diff
@@ -0,0 +1,321 @@
+/* eslint-disable @typescript-eslint/restrict-plus-operands */
+import styled from 'styled-components';
+import dayjs from 'dayjs';
+import type { PickerType } from './types';
+import { classes } from './constants';
+import BaseBox from '~components/Box/BaseBox';
+import getTextStyles from '~components/Typography/Text/getTextStyles';
+import { size } from '~tokens/global';
+import { makeBorderSize, makeSpace } from '~utils';
+import getIn from '~utils/lodashButBetter/get';
+import { useIsMobile } from '~utils/useIsMobile';
+
+const cell = {
+  gap: {
+    x: size[0],
+    y: size[2],
+  },
+  size: {
+    desktop: size[40],
+    mobile: size[48],
+  },
+  background: {
```

Comment:
We have not added any colors for active state? I see the active state is a bit weird color when we are tabbing through the dates

Reply:
Can you share a screenshot not sure what do you mean.